spec/policies/user_policy_spec.rb

```diff
@@ -259,6 +343,41 @@
       end
     end
 
+    context 'for super org admins' do
+      let(:user) { create(:super_org_admin, organisation: org_root_one) }
+
+      it 'includes only super org admins and users belonging to their org' do
```

Comment:
Maybe combine or reword this test and the one following it.  `includes only...` suggests this test tells us the complete set of users that are included, but the next test says `does include...` which sort of breaks that.